python/cudf/cudf/_lib/cpp/stream_compaction.pxd

```diff
@@ -53,3 +60,29 @@ cdef extern from "cudf/stream_compaction.hpp" namespace "cudf" nogil:
         null_equality nulls_equal,
         nan_equality nans_equal,
     ) except +
+
+    cdef unique_ptr[table] stable_distinct(
+        table_view input,
+        vector[size_type] keys,
+        duplicate_keep_option keep,
+        null_equality nulls_equal,
+        nan_equality nans_equal,
+    ) except +
+
+    cdef size_type unique_count(
+        column_view column,
+        null_policy null_handling,
+        nan_policy nan_handling) except +
+
+    cdef size_type unique_count(
+        table_view source_table,
+        null_policy null_handling) except +
```

Comment:
Rather than exposing overloads, I think it'll be cleaner if going forward we only expose the full C++ API assuming no defaults. We have to (or at least, want to) expose all the parameters in Cython, which also means we have to duplicate the defaults, so we'll always be calling the full signature anyway, right?

Reply:
This _is_ the full API, `unique_count` of `table_view`s has no argument for `nan_policy`.

Reply:
See #10183.

Reply:
Ah you are right, never mind. Thanks.